stable/filebeat/templates/daemonset.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         app: {{ template "filebeat.name" . }}
         release: {{ .Release.Name }}
       annotations:
-        checksum/secret: {{ toYaml .Values.config | sha256sum }}
+        checksum/secret: {{ toYaml (default .Values.config .Values.overrideConfig) | sha256sum }}
```

Comment:
going to approve, but I believe it makes it easier to read if written
`        checksum/secret: {{ .Values.overrideConfig | default .Values.config | toYaml | sha256sum }}`